tools/pfe-tools/custom-elements-manifest/ecma-private-class-members.ts

```diff
@@ -0,0 +1,24 @@
+import type { CustomElement } from 'custom-elements-manifest/schema';
+import type { Plugin } from '@custom-elements-manifest/analyzer';
+
+export function ecmaPrivateClassMembersPlugin(): Plugin {
```

Comment:
Ignorant curiosity question:  Is the reasoning for this plugin to supersede what custom-elements-manifest does with the `#method` private hash names identifier?  Or is it just not yet supported?

Reply:
the business end is on line 20, where this plugin sets the privacy to `"private"`